src/Php83/Php83.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php83;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php83
+{
+    private const JSON_MAX_DEPTH = (1 << 31) - 1; // see https://www.php.net/manual/en/function.json-decode.php
+    public static function json_validate(string $json, int $depth = 512, int $flags = 0): bool
+    {
+        if ($flags !== 0 && defined('JSON_INVALID_UTF8_IGNORE') && JSON_INVALID_UTF8_IGNORE !== $flags) {
+            throw new \ValueError('json_validate(): Argument #3 ($flags) must be a valid flag (allowed flags: JSON_INVALID_UTF8_IGNORE)');
+        }
+
+        if ($depth <= 0 ) {
+            throw new \ValueError('json_validate(): Argument #2 ($depth) must be greater than 0');
```

Comment:
The `\ValueError` was introduced by PHP 8.0.

This polyfill should require `symfony/polyfill-php80` or `php>=8` ?

Reply:
Indeed. Bumping to PHP 8 is not an option for now, I think. Let's depend on `symfony/polyfill-php80`.

Reply:
Fixed and rebased - please have another look 👍🏻